plugins/uefi/fu-plugin-uefi.c

```diff
@@ -29,6 +29,12 @@
 #include "fu-plugin.h"
 #include "fu-plugin-vfuncs.h"
 
+/* drop when upgrading minimum required version of fwupdate to 11 */
+#if defined(LIBFWUP_LIBRARY_VERSION)
+#undef  LIBFWUP_VERSION
+#define LIBFWUP_VERSION (LIBFWUP_LIBRARY_VERSION)
```

Comment:
Do we care about LIBFWUP_VERSION at all? Can't we just rely on pkgconfig?

Reply:
That's a good point.  I guess then can use `conf.set_quoted()` too.

Reply:
Fixed